### DISCUSSION TITLE
Redis Extension

### DISCUSSION THREAD

**Discussion by fsteenkamp** [ueberdosis/hocuspocus#107] [Q&A] [answered]
Wondering if the Redis Extension is implemented as a [primary storage](https://www.hocuspocus.dev/guide/documents/#using-a-primary-storage) like RocksDB? Personal preference is for it not to be, I want to use Redis for horizontal scaling and persist somewhere else.

PS. I'm aware of the [re-implementation](https://github.com/ueberdosis/hocuspocus/issues/90#issuecomment-836773215), very excited 😄

**Comment by hanspagel** [comment, marked as answer]
Yes! We’d recommend to combine Redis for horizontal scaling with a hocuspocus worker node writing to RocksDB/Postgres. I’ll put together a guide on scaling soon. ✌️
